### PR TITLE
gremlin: add as/select steps

### PR DESCRIPTION
**Pull request by safchain**
This allows to do a kind of "or"

Now we can merge two results
G.V().Has("Type", "netns").descendants().as("ns").\
  G.V().Has("Type", "host").as("host").\
  select("ns", "host").subgraph()

### CODE REVIEW

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-compile-tests

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Comment by lebauce** [comment]
run skydive-cdd-overview-tests

**Comment by lebauce** [comment]
run skydive-compile-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

File: topology/graph/traversal/traversal.go [hunk 1]
```diff
@@ -711,6 +722,53 @@ func (tv *GraphTraversalV) Sum(keys ...interface{}) *GraphTraversalValue {
...
+func (tv *GraphTraversalV) Select(keys ...interface{}) *GraphTraversalV {
+	if len(keys) == 0 {
+		return &GraphTraversalV{error: errors.New("Select requires at least one key")}
+	}
+
+	ntv := &GraphTraversalV{GraphTraversal: tv.GraphTraversal, nodes: []*graph.Node{}}
+	for _, k := range keys {
+		key, ok := k.(string)
+		if !ok {
+			return &GraphTraversalV{error: errors.New("Select parameter have to be string keys")}
```
Comment (lebauce) [inline topology/graph/traversal/traversal.go]: nit: Select parameters

Comment (lebauce) [reply topology/graph/traversal/traversal.go]: or "Select accepts only string parameters"

File: topology/graph/traversal/traversal.go [hunk 2]
```diff
@@ -810,10 +873,10 @@ func ParseSortParameter(keys ...interface{}) (order common.SortOrder, sortBy str
 		order, ok1 = keys[0].(common.SortOrder)
 		sortBy, ok2 = keys[1].(string)
 		if !ok1 || !ok2 {
-			return order, sortBy, fmt.Errorf("Sort parameters has to be SortOrder(ASC/DESC) and a sort string Key")
+			return order, sortBy, errors.New("Sort parameters has to be SortOrder(ASC/DESC) and a sort string Key")
```
Comment (lebauce) [inline topology/graph/traversal/traversal.go]: nit: have to be

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Pull request merged**